.github/workflows/scripts/parse-wp-version.js

```diff
@@ -0,0 +1,79 @@
+const https = require( 'http' );
+
+module.exports = async function parseWPVersion( wpVersion ) {
```

Comment:
I found this function a bit harder to understand than I would've liked. Maybe a quick comment on top would be helpful:

```suggestion
/**
 * For a given WordPress version finds the URL at which the release ZIP for the 
 * version can be found. For an `X.X` release will find the latest point release by 
 * making a request to the WordPress.org API. 
 *
 * @param {String} The WordPress version for which to find the ZIP file URL: 
 * 'trunk', 'nightly', 'latest', 'X.X', or 'X.X.X'
 * @returns Promise<String> either a URL to the ZIP file containing the requested WordPress version or `WordPress/WordPress#master`.
 */
 module.exports = async function parseWPVersion( wpVersion ) {
```

Also having some tests for this function would make a lot of sense imho.

Reply:
This is a good suggestion. I've written a comment that explains more precisely what this script is for and what it's doing.